devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -224,7 +225,7 @@ def setLevel(level, rate = null) {
 	], 5000)
 }
 
-def configure() {
+def configure(){
```

Comment:
why'd you remove the space here if you're leaving it in the other reformatted functions?

Reply:
There should be also a space here, I will fix that.